css/css-display/display-contents-accessibility.html

```diff
@@ -0,0 +1,321 @@
+    </div>
+
+    <!-- Lists: description lists -->
+    <dl>
+        <dt style="display: contents;" data-expectedrole="term" data-testname="dt with display:contents, as child of dl, has term role" class="ex">q</dt>
+        <dd style="display: contents;" data-expectedrole="definition" data-testname="dd with display:contents, as child of dl, has definition role" class="ex">r</dd>
+        <dt>x</dt>
+        <dd>y</dd>
+    </dl>
+
+    <dl style="display: contents;">
+        <dt style="display: contents;" data-expectedrole="term" data-testname="dt with display:contents, as child of dl with display:contents, has term role" class="ex">q</dt>
+        <dd style="display: contents;" data-expectedrole="definition" data-testname="dd with display:contents, as child of dl with display:contents, has definition role" class="ex">r</dd>
+        <dt>x</dt>
+        <dd>y</dd>
+    </dl>
+
+    <!-- Tables (including grids/treegrids) -->
+    <table style="display: contents;" data-expectedrole="table" data-testname="table with display:contents has table role" class="ex">
+        <thead>
+          <tr style="display: contents;" data-expectedrole="row" data-testname="tr in table with display:contents has row role" class="ex">
+            <th style="display: contents;" data-expectedrole="columnheader" data-testname="td within tr both with display:contents, in table with display:contents, has columnheader role" class="ex">x</th>
+            <th>x</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td style="display: contents;" data-expectedrole="cell" data-testname="td within tr in table, all with display:contents, has cell role" class="ex">x</td>
+            <td>x</td>
+          </tr>
+        </tbody>
+      </table>
+
+    <div role="table" style="display: contents;" data-expectedrole="table" data-testname="div with role table with display:contents has table role" class="ex">
+        <div role="rowgroup" style="display: contents;" data-expectedrole="rowgroup" data-testname="div with role rowgroup in div with role table, both with display:contents, has rowgroup role" class="ex">
+            <div role="row">
+                <div role="columnheader" style="display: contents;" data-expectedrole="columnheader" data-testname="div with role columnheader with display:contents, in div with role rowgroup within div with table role both with display:contents, has columnheader role">x</div>
+                <div role="columnheader">x</div>
+            </div>
+        </div>
+        <div role="rowgroup">
+            <div role="row">
+                <div role="cell"></div>
+                <div role="cell" style="display: contents;" data-expectedrole="cell" data-testname="div with cell role and display:contents, within div with role row contained in a div with role table both with display:contents, has cell role" class="ex"></div>
```

Comment:
This Chromium failure looks legit to me, but again given the error message (`!EQ("cell", "gridcell")`) probably not specific to `display:contents`. 

I'd remove this one. Test results indicate Chromium's `cell` is probably implemented as a synonym of `gridcell`, and that's already tested (and failing) in [`wai-aria/role/roles.html`](https://wpt.fyi/results/wai-aria/role/table-roles.html?label=master&label=experimental&aligned&q=label%3Aaccessibility).

Reply:
@cookiecrook Done: https://github.com/web-platform-tests/wpt/pull/43740/commits/ce29a7f89cc670f87a957bafda621559558c1aed.